src/main/java/com/playtika/homework2/task2/ManipulationsWithListOfStudent.java

```diff
@@ -0,0 +1,93 @@
+package com.playtika.homework2.task2;
+
+import java.util.*;
+
+public class ManipulationsWithListOfStudent {
+
+    public static void main(String[] args) {
+        int[] id = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+        String[] firstName = {"Андрей", "Павел", "Евгений", "Анастасия", "Юрий", "Ольга", "Александр", "Артем", "Денис", "Алиса"};
+        String[] lastName = {"Попков", "Жуковский", "Невмержицкий", "Бартош", "Хизюк", "Соболева", "Кондратович", "Петров", "Петров", "Иванова"};
+        int[] age = {21, 19, 22, 18, 17, 22, 19, 20, 20, 23};
+        List<Student> students = new ArrayList<>();
```

Comment:
use list instead of arrays:
List students = new ArrayList<>();
students.add(new Student(...));
...